### PR TITLE
MWC7K 2018 Finals update

### PR DESCRIPTION
**Pull request by juankristal**
Only bracket missing, should be uplodaded during the week.

Finished; Pending Review.

### CODE REVIEW

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 1]
```diff
@@ -56,31 +56,19 @@ The osu!mania 7K World Cup 2018 is run by various community members by distribut
 
 ------------------------------------------------------------------------
 
-## Match Schedule: Semifinals
+## Match Schedule: Finals
 
-### Saturday, 3. March 2018
+### Sunday, 11. March 2018
```
Comment (TPGPL) [inline wiki/Tournaments/MWC/2018/7K/en.md]: 2018-03-11

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 2]
```diff
@@ -129,6 +117,29 @@ The osu!mania 7K World Cup 2018 is run by various community members by distribut
 
 ## Mappools
 
+### Finals
+
+**[Download the mappack here! (105 MB)](https://mega.nz/#!QvghCDCQ!mDHFhurA3mzdqxD-2bfULBNG0cBtXzmG35T-hw6fa3o)**
+
+- Freemod
```
Comment (TPGPL) [inline wiki/Tournaments/MWC/2018/7K/en.md]: FreeMod

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 3]
```diff
@@ -129,6 +117,29 @@ The osu!mania 7K World Cup 2018 is run by various community members by distribut
...
+  - [Chroma - Tiny Tales Continue (qodtjr) \[Recollections\]](http://osu.ppy.sh/b/1421269&m=3) 
+  - [Camellia feat. Nanahira - EDM Jumpers ({E+H}DM Reboot) (Evening) \[* * * \ ( o w o)/ ** !!\]](http://osu.ppy.sh/b/1309991&m=3)
```
Comment (TPGPL) [inline wiki/Tournaments/MWC/2018/7K/en.md]: Escape the stars and brackets

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 4]
```diff
@@ -209,6 +220,22 @@ The osu!mania 7K World Cup 2018 is run by various community members by distribut
 
 ## Match Results
 
+### Semifinals
+
+| Saturday, 3. March 2018 | | | | |
```
Comment (TPGPL) [inline wiki/Tournaments/MWC/2018/7K/en.md]: 2018-03-03

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 5]
```diff
@@ -209,6 +220,22 @@ The osu!mania 7K World Cup 2018 is run by various community members by distribut
...
+| United States ![][flag_US] | 6 | 5 | ![][flag_JP] Japan | [#1](https://osu.ppy.sh/community/matches/40519477) |
+| Malaysia ![][flag_MY] | 0 | 6 | ![][flag_KR] South Korea | [#1](https://osu.ppy.sh/community/matches/40527504) |
+| China ![][flag_CN] | 6 | 0 | ![][flag_SG] Singapore | [#1](https://osu.ppy.sh/community/matches/40528604) |
+| Italy ![][flag_IT] | 4 | 6 | ![][flag_PH] Philippines | [#1](https://osu.ppy.sh/community/matches/40530352) |
+| France ![][flag_FR] | 0 | 6 | ![][flag_BR] Brazil | -WIN BY DEFAULT- |
+| Brazil ![][flag_BR] | 6 | 1 | ![][flag_US] United States | [#1](https://osu.ppy.sh/community/matches/40544245) |
+
+| Sunday, 4. March 2018 | | | | |
+| ---: | :---: | :---: | :--- | :---: |
+| Chile ![][flag_CL] | 2 | 6 | ![][flag_PH] Philippines | [#1](https://osu.ppy.sh/community/matches/40554885) |
```
Comment (WalterToro) [inline wiki/Tournaments/MWC/2018/7K/en.md]: This was 3-6, not 2-6

File: wiki/Tournaments/MWC/2018/7K/en.md (hunk 1 above)
Comment (juankristal) [reply wiki/Tournaments/MWC/2018/7K/en.md]: surprised that his is pointed out on the finals but not on previous rounds 🤔

Comment (TPGPL) [reply wiki/Tournaments/MWC/2018/7K/en.md]: ye should on all of them, forgot about it 🤔

File: wiki/Tournaments/MWC/2018/7K/en.md (hunk 2 above)
Comment (TPGPL) [reply wiki/Tournaments/MWC/2018/7K/en.md]: same on all of them

**Pull request merged**